services.yaml

```diff
@@ -245,7 +245,53 @@ paths:
                 items:
                   $ref: "#/definitions/Deployment"
         401:
-          $ref: "#/responses/UnauthorizedError"
+            $ref: "#/responses/UnauthorizedError"
+  /services/{env_id}/overops-status/agents:
```

Comment:
we need another overops-status/ main GET that has both agents and collectors in the response

Reply:
what about just 'overops-status'?

Reply:
yes

overops-status
overops-status/agents
overops-status/collectors